acceptance-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+docker run -d -h oauth --name oauth -t danpersa/rusty-oauth:latest
+docker run -d -h postgres -e POSTGRES_PASSWORD= -e POSTGRES_USER=innkeepertest --name db -t postgres:9.4
+sbt  assembly
```

Comment:
- removed the space
- I start them  before to be sure they have time to start... the first thing innkeeper does when it starts, is creating the schema... if the postgres is not up, it will fail...